satellite/satellitedb/dbx/satellitedb.dbx.postgres.sql

```diff
@@ -188,10 +188,11 @@ CREATE TABLE users (
 	PRIMARY KEY ( id )
 );
 CREATE TABLE value_attributions (
-	bucket_id bytea NOT NULL,
+	project_id bytea NOT NULL,
+	bucket_name bytea NOT NULL,
 	partner_id bytea NOT NULL,
 	last_updated timestamp NOT NULL,
-	PRIMARY KEY ( bucket_id )
+	PRIMARY KEY ( project_id, bucket_name )
```

Comment:
It is possible for one partner to have multiple buckets. In that case can you make multiple entries for the same partner id ?